openhands/llm/metrics.py

```diff
@@ -1,12 +1,24 @@
+import time
+
+from pydantic import BaseModel, Field
+
+
+class Cost(BaseModel):
+    model: str
+    cost: float
+    timestamp: float = Field(default_factory=time.time)
+
+
 class Metrics:
     """Metrics class can record various metrics during running and evaluation.
     Currently, we define the following metrics:
         accumulated_cost: the total cost (USD $) of the current LLM.
     """
 
-    def __init__(self) -> None:
+    def __init__(self, model_name: str = 'default') -> None:
```

Comment:
Do we want to have a default here? We could also make it required to make sure that it gets recorded.

Reply:
ok.. i think we should leave a default value because the `State` may need to construct a copy of it.